src/serial/private/serialport/serialport_common.nim

```diff
@@ -27,17 +27,17 @@ type
     Two = 2,
     OnePointFive = 3
 
-  InvalidSerialPortError* = object of Exception
+  InvalidSerialPortError* = object of IOError
 
   TimeoutError* = object of IOError
 
   InvalidSerialPortStateError* = object of IOError
 
-  InvalidBaudRateError* = object of Exception
+  InvalidBaudRateError* = object of ValueError
 
-  InvalidDataBitsError* = object of Exception
+  InvalidDataBitsError* = object of ValueError
 
-  InvalidStopBitsError* = object of Exception
+  InvalidStopBitsError* = object of IOError
```

Comment:
This should maybe be `ValueError` too?

Reply:
right, I'm not sure why I chose `IOError` in the first place.